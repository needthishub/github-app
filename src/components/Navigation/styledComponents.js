import styled from 'styled-components';
import {Link} from 'react-router-dom';

export const Container = styled.div`
  display: flex;
  align-items: center;
  background-color: black;
  padding: 10px;
  width: 100%;
  margin-bottom: 15px;
`;

export const Title = styled.h1`
  color: white;
  font-size: 18px;
`;

export const Menu = styled(Link)`
  color: white;
  text-decoration: none;
  margin-left: 15px;
`;
