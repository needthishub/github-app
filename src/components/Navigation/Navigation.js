import React from 'react';
import {Container, Title, Menu} from './styledComponents';

function Navigation() {
  return (
    <Container>
      <Title>Github Search</Title>
      <Menu to="/">Search</Menu>
      <Menu to="/about">About</Menu>
    </Container>
  );
}

export default Navigation;
