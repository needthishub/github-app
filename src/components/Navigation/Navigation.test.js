import React from 'react';
import {shallow} from 'enzyme';
import Navigation from './Navigation';

describe('<Navigation/>', () => {
  it('should match to snapshot', () => {
    const wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
