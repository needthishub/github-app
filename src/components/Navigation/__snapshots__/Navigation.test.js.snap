// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation/> should match to snapshot 1`] = `ShallowWrapper {}`;
